rocAL/rocAL_pybind/amd/rali/readers.py

```diff
@@ -0,0 +1,101 @@
+import rali_pybind as b
+from amd.rali.pipeline import Pipeline
+
+def coco(*inputs,file_root, annotations_file='', bytes_per_sample_hint=0, dump_meta_files=False, dump_meta_files_path='', file_list='', initial_fill=1024,  lazy_init=False, ltrb=False, masks=False, meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1,
+                 preserve=False, random_shuffle=False, ratio=False, read_ahead=False,
+                 save_img_ids=False, seed=-1, shard_id=0, shuffle_after_epoch=False, size_threshold=0.1,
+                 skip_cached_images=False, skip_empty=False, stick_to_shard=False, tensor_init_bytes=1048576):
+
+    Pipeline._current_pipeline._reader = "COCOReader"
+    #Output
+    labels = []
+    bboxes = []
+    kwargs_pybind = {"source_path": annotations_file, "is_output":True}
+    meta_data = b.COCOReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (meta_data, labels, bboxes)
+
+def file(*inputs, file_root, bytes_per_sample_hint=0, file_list='', initial_fill='', lazy_init='', num_shards=1,
+                 pad_last_batch=False, prefetch_queue_depth=1, preserve=False, random_shuffle=False, read_ahead=False,
+                 seed=-1, shard_id=0, shuffle_after_epoch=False, skip_cached_images=False, stick_to_shard=False, tensor_init_bytes=1048576, device=None):
+
+    Pipeline._current_pipeline._reader = "labelReader"
+    #Output
+    labels = []
+    kwargs_pybind = {"source_path": file_root}
+    label_reader_meta_data = b.labelReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
```

Comment:
what is label_reader_meta_data and where is it used?

Reply:
To be aligned with the DALI's outer API, we return an empty list as currently out meta_data_reader does not return anything

label_reader_meta_data makes a call to the raliCreateLabelReader